src/lib/util.js

```diff
@@ -69,3 +69,11 @@ exports.formatMs = function(ms) {
 exports.removeTrailingSlash = function(str = '') {
   return str.endsWith('/') ? str.substr(0, str.length - 1) : str;
 };
+
+exports.streamToString = async function streamToString(readable) {
```

Comment:
Can we keep the test helpers out of the main functions? To keep the packed size small, we should avoid publishing tests & other dev specific files.

* Create a separate dir `/src/test-helper/index.js`
* To exclude `test-helper` from publishing, add the following line to the `package.json`s `files` field.

```
"files": [
    ...,
    !src/test-helper
  ],
```